dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM php:7-cli
```

Comment:
Would you mind explicitly defining it as 7.4? It doesn't make a different but with the amount of change between 7.0 - 7.4 i'd rather be specific